src/otk/command.py

```diff
@@ -81,13 +85,11 @@ def _process(arguments: argparse.Namespace, dry_run: bool) -> int:
     if kind != "osbuild":
         raise ValueError("only target osbuild supported right now")
     spec = OSBuildContext(ctx)
-    target = OSBuildTarget()
-    state = State(path=path)
-    tree = resolve(spec, state, doc.tree[f"{PREFIX_TARGET}{kind}.{name}"])
+    tree = resolve(spec, State(path=cwd), doc.tree[f"{PREFIX_TARGET}{kind}.{name}"])
```

Comment:
Unrelated to this PR - I think this will not work when the `target.osbuild.foo` is in a subdirectoy. I.e. if the manifest looks like this:
```
otk.include "subdir/osbuild.otk"
---
subdir/osbuild.otk:

otk.target.osbuild.foo:
  otk.include "file-in-subdir.yaml"
```
the code will not be able to find `file-in-subdir.yaml` because it points to the current cwd. 

This is already a problem and unrelated to the PR. I think the solution would be to do a single tree resolving and pass the expected target(s) into resove() so that resolve() can just skip over anything otk.target.* that is not part of "target_requested" but this way we would have a complete tree in a single run.

Reply:
partly unrelated - so I did a second PR
https://github.com/osbuild/otk/pull/158
but this also works when rebased on this stdin PR 🥳
all good 😏

Reply:
Sorry, I think I did not express my point very well :( Thanks for writing the test! I added a small tweak the test PR in https://github.com/osbuild/otk/pull/158 and it is now failing  and then I looked a bit more in https://github.com/osbuild/otk/pull/159 that also includes a fix now (not 100% happy yet with #159 but I hope/think it demonstrates the idea(s))